src/antsibull/changelog.py

```diff
@@ -320,9 +321,9 @@ async def download_changelog(
         self.changelog = ChangelogData.concatenate(changelogs)
 
     async def download_porting_guide(self, aio_session: aiohttp.client.ClientSession):
-        branch_url = "https://raw.githubusercontent.com/ansible/ansible/devel"
-
-        query_url = f"{branch_url}/{get_porting_guide_filename(self.latest)}"
+        query_url = (
+            f"{CORE_PORTING_GUIDE_BASE_URL}/{get_porting_guide_filename(self.latest)}"
+        )
```

Comment:
Hmm, this code is wrong in the sense that it uses the wrong version of the porting guide, except in very specific lucky situations. But then, the code was already doing this wrong before the docs were moved... We have to fix this properly at some point, but for now this is ok.

Reply:
Yeah, right. I wanted to keep this as a Minimal Changes To Make It Work Like It Did Before PR. Along with this, there is some other stuff like residual ansible-base references that need cleanup.

Reply:
In that case the constant should not move to another file, but stay in this local function.